students/JerryH/Lesson07/html_render.py

```diff
@@ -0,0 +1,120 @@
+class Element:
+
+    # Class Attributes
+    tag = ""
+    indent = "  "
+
+    def __init__(self, content=None, **style):
+        self.content = [content] if content else []
+        self.styles = style
+
+
+    def append(self, words):
+        self.content.append(words)
+
+
+    def render(self, fileout, cur_ind=""):
```

Comment:
styling suggestions: class methods have 1 space in between (while non-class functions have 2 spaces)